### PR TITLE
Allow user to add feedback_url via themes functions.php. Fixes #598

### LINKED ISSUE #598: [Feature Request]: Support feedback_url in meeting feed
### Contact Details

_No response_

### Requested Feature/Enhancment

TSML-UI supports the recently documented feedback_url field here:

https://github.com/code4recovery/spec/pull/28/files

Create a new field in the meeting editor and incorporate this new meta field in to the JSON feed for TSML-UI users.

Secondary features:

Incorporate into legacy UI
Append meeting name to query stringPlease add this cool feature!

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### PR DESCRIPTION
**Pull request by 97eighty**
For advanced users using tsml_ui. Allows user to build a URL to be included in meetings feed.

$tsml_feedback_url = "https://domain.com?meeting={{slug}}";
$tsml_feedback_url = "https://domain.com?meeting={{id}}";
$tsml_feedback_url = "mailto:office@domain.com?subject={{slug}}";

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -91,6 +91,12 @@ Add this to your theme's functions.php.
 
 	$tsml_street_only = false;
 
+= Can I add a feedback_url to each meeting when using tsml_ui? = Add a URL to your themes functions.php.
```
Comment (joshreisner) [inline readme.txt:94]: it would be good to refer to it as `TSML UI` for consistency

File: includes/functions.php [hunk 2]
```diff
@@ -1047,6 +1047,25 @@ function tsml_get_meeting($meeting_id = false)
...
+    if (isset($tsml_feedback_url)) {
+        $id = $post->ID;
+        $slug = $post->post_name;
+
+        $url = $tsml_feedback_url;
+
+        $url = str_replace('{{id}}', $id, $url);
+        $url = str_replace('{{slug}}', $slug, $url);
+    }
+    return esc_url_raw($url, ['http', 'https', 'mailto', 'tel', 'sms']);
```
Comment (joshreisner) [inline includes/functions.php:1065]: `$url` might not be defined, i think this will throw a warning for users who aren't suppressing them

Comment (joshreisner) [inline includes/functions.php:1065]: i don't think we actually want to escape the URL for the feed, this will encode `&` as `&amp;`

**Review by joshreisner — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 1 above)
Comment (97eighty) [reply readme.txt]: Good point. Change made.

File: includes/functions.php (hunk 2 above)
Comment (97eighty) [reply includes/functions.php]: Modified this function.

**Review by joshreisner — APPROVED** [review]

**Pull request merged**